custom-login/src/app/login/login.component.ts

```diff
@@ -17,6 +17,11 @@ import sampleConfig from '../app.config';
 
 const DEFAULT_ORIGINAL_URI = window.location.origin;
 
+let USE_INTERACTION_CODE = false;
+if (sampleConfig.oidc.useInteractionCodeFlow === 'true') {
```

Comment:
it would be good to ensure that this property is a `boolean` value (see comment on app.config.ts)